### PR TITLE
Fix credential deletion.

### PR DESCRIPTION
**Pull request by mattcollier**

### CODE REVIEW

File: credential-viewer-component.js [hunk 1]
```diff
@@ -141,13 +141,20 @@ function Ctrl(
   };
 
   self.confirmDeleteCredential = function(err, result) {
+    var credentialServiceOptions = {};
+    if(!compareHost(self.credential.id)) {
+      credentialServiceOptions.url = self.altUpdateEndpoint + '?id=' +
+        encodeURIComponent(self.credential.id);
+    }
```
Comment (dlongley) [inline credential-viewer-component.js:148]: I'd prefer for us to get rid of this magic somehow, but don't know how to yet, so it's fine for now.

File: credential-viewer-component.js [hunk 2]
```diff
@@ -141,13 +141,20 @@ function Ctrl(
...
     if(!err && result === 'ok') {
       self.credential.deleted = true;
-      // wait to delete so modal can transition
-      brCredentialService.collection.del(self.credential.id, {delay: 400})
+      brCredentialService.collection
+        .del(self.credential.id, credentialServiceOptions)
```
Comment (dlongley) [inline credential-viewer-component.js:152]: This removed the delay, did we need that? I don't remember it's purpose despite the comment.

**Comment by dlongley** [comment]
Just one question to answer, then LGTM, merge.

File: credential-viewer-component.js (hunk 2 above)
Comment (mattcollier) [inline credential-viewer-component.js:152]: Discussed in IRC, removing the delay seems to have no impact.

**Pull request merged**